react/src/components/toggle/SprkToggle.js

```diff
@@ -99,6 +112,10 @@ SprkToggle.propTypes = {
   titleAddClasses: PropTypes.string,
   /** Additional classes for the toggle icon. */
   iconAddClasses: PropTypes.string,
+  /** A string that is used to set the aria-controls and associated id for the component. 
+   * The default is generated using lodash.uniqueId.
```

Comment:
We don't need to mention that we are using lodash specifically to generate the ID. All the user needs to know is that there is a default generated if they don't provide one.

Reply:
That was something I added after I saw that the documentation shows the default value as `uniqueId('sprk_toggle_content_')` but I can certainly take it back out.

Reply:
oh hmm. I still think that they probably dont care how the ID is generated or with what. They'll just assume thats a function we use.